src/common/service/NetRemoteDataStreamingService.cxx

```diff
@@ -36,3 +37,14 @@ NetRemoteDataStreamingService::DataStreamBidirectional([[maybe_unused]] grpc::Ca
 
     return std::make_unique<Reactors::DataStreamReaderWriter>().release();
 }
+
+grpc::ServerUnaryReactor*
```

Comment:
I guess this has to return a reactor due to the `NetRemoteDataStreaming` impl type?

Reply:
Yeah since I'm using the gRPC callback API, then the unary RPCs on the server side use `ServerUnaryReactor`. There is a bit more info here: https://github.com/grpc/proposal/blob/master/L67-cpp-callback-api.md